procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,88 @@
+# Council Synchronous Meetings
+
+This documents procedures, norms, and logistics of the Council's sync meetings.
+
+## Frequency
+
+The Council meets once a week.
+
+After the 14th of July, 2023, the Council will re-evaluate whether this frequency still makes sense or whether it should be changed.
+
+## Location
+
+The meeting happens in some video call software (e.g., Google Meet).
+
+## Quorum
+
+The meeting has reached quorum if at *most* 2 Council representatives (and their alternates) are not present (i.e., N - 2).
```

Comment:
This is probably a different policy question, but have we talked about how we pick alternates? Are alternates a standing position, or is this something the council member can appoint as needed?

Reply:
I believe the RFC indicates that the top-level team is responsible for choosing an alternate ([source](https://rust-lang.github.io/rfcs/3392-leadership-council.html#alternates-and-forgoing-representation)).

So, I'd say the selection process is up to the team. And whether they select ahead-of-time, or on an as-needed basis is also presumably up to the team.

Presumably if a council member knows they are going to be unavailable, they are responsible for letting their team know and get the team to pick someone. If the council member is becomes unavailable without communicating, the council should inform the team.

I'm personally unsure if there should be standing alternates or not. It seems like it might be a good idea just so there isn't a scramble to pick someone. But I could imagine people may not want to be on the hook with uncertainty if they will actually ever be needed.

Reply:
Yes, this was something the governance reform working group explicitly did not want to dictate or have the Council dictate to the top-level teams since it is their responsibility to select representatives how they see fit. However, we *could* offer "best practices" for teams to adopt which they may deviate from if they wish to. Thoughts?

Reply:
If it's meant to be a softer recommendation, I'd prefer to not have it firmly documented, as what we put here will carry weight that may be incorrectly used.

Reply:
I agree with the idea of leaving things up to the team where possible. Having some suggestions, like "we recommend teams do X but teams are free to make their own choices based on their needs" would be okay with me, but I think it makes sense to hold off on making these recommendations for now.

Reply:
Going to resolve this as it's not really relevant to this document. Happy to see this logged somewhere else though.